src/extension.ts

```diff
@@ -82,6 +82,8 @@ export const activate = async (context: vscode.ExtensionContext) => {
   })
 
   const status = await cliStatus()
+  const auth0UserId = status.a0UserId
+  StateManager.setState(KEYS.AUTH0_USER_ID, auth0UserId)
 
```

Comment:
we should probably move this into the login command, since that's where it may change